i18n/zh-cn/docusaurus-plugin-content-docs/current/quick-start/opensergo-control-plane.md

```diff
@@ -12,12 +12,34 @@ sidebar_position: 1
 
 ## 在集群中安装 OpenSergo 控制平面
 
-TODO: helm chart
+```shell
+# 创建 opensergo-system 命名空间
+kubectl apply -f opensergo-control-plane/k8s/namespace.yaml
+
+# opensergo 控制面相关权限配置
+kubectl apply -f opensergo-control-plane/k8s/rbac/role.yaml
+kubectl apply -f opensergo-control-plane/k8s/rbac/rolebinding.yaml
+kubectl apply -f opensergo-control-plane/k8s/rbac/serviceaccounts.yaml
```

Comment:
need update to match new yaml file 
<img width="521" alt="image" src="https://user-images.githubusercontent.com/43714056/211733678-6d90f864-5dd7-4380-8491-28788f7351eb.png">